docs/second_factor_auth.md

```diff
@@ -0,0 +1,54 @@
+# Second Factor Authentication (2FA)
+
+According with [wikipedia](https://en.wikipedia.org/wiki/Multi-factor_authentication) Two-factor authentication (also known as 2FA) is a type (subset) of multi-factor authentication. It is a method of confirming a user's claimed identity by utilizing a combination of two different factors: 1) something they know, 2) something they have, or 3) something they are.
+
+This feature provider 2FA for OpenStack Keystone based on email.
```

Comment:
Typo: provider -> provides

Reply:
Fixed in: https://github.com/telefonicaid/fiware-keystone-spassword/pull/57/commits/d97ef84d0e7f433fe8eac697c526156bf89648b1